### PR TITLE
Problem: we don't have enough packages

### PR DESCRIPTION
**Pull request by yrashk**
Solution: import the bulk of packages from Pigsty

For now, we'll select all packages that are on github or gitlab. Some of them will build, some won't. But we'll know better once we try.

### CODE REVIEW

File: pigsty.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+URI.open("https://raw.githubusercontent.com/pgsty/extension/refs/heads/main/data/pigsty.csv") do |f|
+  i = 0
+  CSV.foreach(f, headers: true) do |row|
+    next unless row["url"] =~ /github/ || row["url"] =~ /gitlab/
+
+    origin = row["url"]
+    if row["url"] =~ /github/
+      github = origin.match(%r{https://github\.com/([^/]+)/([^/]+)}).captures.join("/")
+      origin = "github \"#{github}\""
```
Comment (jonatas) [inline pigsty.rb:20]: Idea: group by github workspace to also detect workspaces for cases like omnigres/timescale.

**Review by jonatas — APPROVED** [review]
This looks so smart!

We may also think about having pigsty as a "source provider" and create a concept of dynamic packages from source providers, but in this way we can import and start fixing case by case looks much better for long term.

File: pigsty.rb (hunk 1 above)
Comment (yrashk) [reply pigsty.rb:20]: 🤔 Maybe we can do this on a case-by-case scenario. Like: should we do this for personal github accounts? What should we do for non-github projects?

**Comment by yrashk** [comment]
The ultimate dream is that this will help @Vonng bring in and maintain packages for Pigsty's distributions, eliminating the tedious work. 

However, as a backup, we can look into "dynamic package sources" where feasible, where we can translate a formal build definition into our format. I like this idea.

**Pull request merged**